src/ensembl/src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewState.ts

```diff
@@ -51,21 +59,75 @@ export enum GeneFunctionTabName {
 
 export enum GeneRelationshipsTabName {
   ORTHOLOGUES = 'Orthologues',
-  PARALOGUES = 'Paralogues',
   GENE_FAMILIES = 'Gene families',
-  GENE_CLUSTERS = 'Gene clusters',
-  GENE_PANELS = 'Gene panels',
-  GENE_NEIGHBOUTHOOD = 'Gene neighbouthood',
-  GENE_SIMILARITY = 'Gene similarity within-genome'
+  GENE_PANELS = 'Gene panels'
 }
 
-export const defaultEntityViewerGeneViewUIState = {
-  selectedGeneTabName: GeneViewTabName.TRANSCRIPTS,
-  geneFunction: {
-    selectedTabName: null
-  },
-  geneRelationships: {
-    selectedTabName: null
+export type GeneViewTabData = {
+  view: View | '';
+  primaryTab: GeneViewTabName;
+  secondaryTab: GeneFunctionTabName | GeneRelationshipsTabName | null;
+};
+
+export const transcriptsTabData: GeneViewTabData = {
+  view: '',
+  primaryTab: GeneViewTabName.TRANSCRIPTS,
+  secondaryTab: null
+};
+
+// using Map to guarantee the order of the inserted elements
+export const GeneViewTabMap: Map<View, GeneViewTabData> = new Map();
+GeneViewTabMap.set(View.PROTEIN, {
+  view: View.PROTEIN,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.PROTEINS
+});
```

Comment:
Any chance we can include the tab display name here?
By that we can simplify this:
https://github.com/Ensembl/ensembl-client/pull/309/files#diff-a8e10f9cafccfd34f324bb947c8c9edfR63-R69

Reply:
My idea for symplifying [this](https://github.com/Ensembl/ensembl-client/blob/b1a6903bed2c43e7d36c096cc207aa6fd4f4ecd1/src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-relationships/GeneRelationships.tsx#L63-L68) was to separate the `title` property of the Tab ([link](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/tabs/Tabs.tsx#L24)) into a label and a value, like we talked about. In which case the `onClick` callback can be called with the actual value of the `view` rather than with the display name of the tab. But I thought we agreed to do this in a separate PR.

Reply:
Yes, I was just suggesting this as a temporary solution but let's leave it as it is for now.